src/diff.ts

```diff
@@ -90,3 +90,8 @@ export interface Diff {
   copied: string[];
   modified: FileDiff[];
 }
+
+export type EofNewLine =
+    "oldMissing"
+  | "newMissing"
+  | "bothMissing";
```

Comment:
Maybe an enum is a bit nicer?

https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
Ah yes you are right, forgot about the string enums.. 👍 wanted to handle camelCase serialization that we eventually get from the http-api